test/helpers/getComponentName.js

```diff
@@ -16,6 +17,20 @@ describe('getComponentName', () => {
     expect(getComponentName(Foo)).to.equal(Foo.displayName);
   });
 
+  it('given a memo, returns the name or displayName', () => {
```

Comment:
Not every tested React version has React.memo available.
```suggestion
  (React.memo ? it : it.skip)('given a memo, returns the name or displayName', () => {
```

Reply:
i'm actually super confused how this test passed on React 15 and 0.14.

Reply:
turns out there's something wrong with the travis.yml; doesn't have to be a blocker here.